salesforce_api/gsheet-new-row-to-sfdc-new-contact/sample.bal

```diff
@@ -0,0 +1,78 @@
+import ballerina/log;
+import ballerinax/googleapis.sheets;
+import ballerinax/salesforce as sfdc;
+
+// Types
```

Comment:
No need of this comment.

Reply:
Updated in [86c29e1](https://github.com/ballerina-guides/integration-samples/pull/31/commits/86c29e1236dcc64dd4c790e62e2633241a8a8307)